src/gutter-component.coffee

```diff
@@ -15,13 +15,26 @@ GutterComponent = React.createClass
   dummyLineNumberNode: null
   measuredWidth: null
 
+  propTypes:
+    isHiddenInitially: React.PropTypes.bool
+
+  getDefaultProps: ->
+    isHiddenInitially: false
```

Comment:
After my PR lands, I'd like to store all *essential* state associated with the component in the model and all *derived* state in the presenter. Ideally the only `props` any view component should have in the editor component hierarchy is the presenter. In this case, I think you can base the visibility of the gutter component on the model for now.